config/manifests/bases/gatekeeper-operator.clusterserviceversion.yaml

```diff
@@ -13,7 +13,10 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Gatekeeper is the Schema for the gatekeepers API
+    - description: Gatekeeper is the Schema for the gatekeepers API. Gatekeeper contains
```

Comment:
Seems like these are similar descriptions from the `operator.gatekeeper.sh_gatekeeperrs.yaml. 

If that is accurate, I suggestion similar feedback